src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTest.php

```diff
@@ -548,7 +548,7 @@ private function assertUrlPackage(ContainerBuilder $container, DefinitionDecorat
     private function assertVersionStrategy(ContainerBuilder $container, Reference $reference, $version, $format)
     {
         $versionStrategy = $container->getDefinition($reference);
-        if (null === $version) {
+        if (null === $version || !$versionStrategy instanceof DefinitionDecorator) {
```

Comment:
When is `$versionStrategy` allowed to be an instance of `DefinitionDecorator` while `$version` is `null`?